src/markd/parsers/block.cr

```diff
@@ -122,7 +123,7 @@ module Markd::Parser
         # this is a little performance optimization
         unless @indented
           first_char = @line[@next_nonspace]?
-          unless first_char && (Rule::MAYBE_SPECIAL.includes?(first_char) || first_char.ascii_number?)
+          unless first_char && (Rule::MAYBE_SPECIAL.includes?(first_char) || first_char.ascii_number? || @line.match(/(?<!\\)\|/))
```

Comment:
Tables should only be parsed when `@options.gfm` is true

Reply:
Ok, that was actually pretty easy. Done in latest commit.